signature-specification.md

```diff
@@ -248,6 +249,47 @@ The signing time denotes the time at which the signature was generated. A X509 c
+1. *Private Artifacts* - Artifacts that are private to a user or organization, and may be shared with limited parties. 
+E.g. Images for containerized applications and services used within an organization, or shared with limited authorized parties. 
+These applications and services may be either intended to run in any infrastructure or may run on specific infrastructure e.g. integration with a cloud service provider. 
+Therefore, the signature’s portability should at least as much as the artifact’s portability.
+
+*Notary v2 signature portability* is based on the following
+
+**Signature discovery**
+
+Notary v2 addressed signature discovery by storing signatures in the same registry (location) where an artifact is present.
+This is currently supported through [ORAS artifact spec](https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md) which allows reference artifacts such as signatures, SBOMs to be associated with existing artifacts like Images.
+Notary v2 allows multiple signatures to be associated with an artifact, and clients may automatically push signatures for an artifact to a destination registry when a signed artifact moves from one registry to other.
+**TBD** Notation may allow users to filter certain signatures if they are meant for private usage and not for public consumption.
+
+**Verification requirements**
+
+Notary v2 supports a range of signed artifacts intended for public and private distribution.
+Signatures generated by Notary v2 without extended signature attributes marked critical can be verified in any environment where Notation client or another Notary v2 standards compliant verification tool is available, without any additional dependencies.
+It should be noted that revocations checks, which usually relies on an external mechanism such as CRL/OCSP may require the verification environment to have access to local network or public internet, to have access to the CRL/OCSP endpoint.
+
+Notary v2 also supports signatures generated using compliant signing plugins, which allow vendors to optionally provide additional features on top of Notary v2 standard features.
+Verification of these signatures may require additional dependencies like Notary v2 compliant verification plugin, making these signatures more appropriate to use where broad portability may not be required for the associated signed artifact.
+This allows users to implement security controls required for their organizations, that are not broadly applicable and may take time to standardize.
+E.g. Integration with a signature transparency log as part of signature verification.
+
+Based on user’s requirements, a user can select appropriate signing mechanism that produces signatures with desired portability.
+Notation generated signatures using local keys and signing plugins of type *Raw Signature Generator* always produce signatures which can be verified without any additional dependencies.
```

Comment:
Does "local" here refer to local keys on the node doing the signing? Or, local to a cloud provider/isv where the actual private key is stored securely in an ISV keyvault?

Reply:
I've simplified the language used here without referring local keys or signing plugin type.